app/gui/src/dashboard/components/dashboard/column/LabelsColumn.tsx

```diff
@@ -19,21 +19,29 @@ import ManageLabelsModal from '#/modals/ManageLabelsModal'
 
 import * as backendModule from '#/services/Backend'
 
+import { useStore } from '#/hooks/storeHooks'
+import { useDriveStore } from '#/providers/DriveProvider'
 import * as permissions from '#/utilities/permissions'
+import { EMPTY_ARRAY } from 'enso-common/src/utilities/data/array'
 
 // ====================
 // === LabelsColumn ===
 // ====================
 
 /** A column listing the labels on this asset. */
 export default function LabelsColumn(props: column.AssetColumnProps) {
-  const { item, state, rowState } = props
+  const { item, state } = props
   const { backend, category, setQuery } = state
-  const { temporarilyAddedLabels, temporarilyRemovedLabels } = rowState
   const { user } = authProvider.useFullUserSession()
   const { setModal, unsetModal } = modalProvider.useSetModal()
   const { getText } = textProvider.useText()
-  const { data: labels } = backendHooks.useBackendQuery(backend, 'listTags', [])
+  const { data: labels } = backendHooks.useBackendQuery(backend, 'listTags', EMPTY_ARRAY)
```

Comment:
I think we're playing too much with micro optimizations here, not sure that `EMPTY_ARRAY` is somewhat better that just `[]`. what if we introduce an option for this query? With just literals we'd add an argument.